app/controllers/home_controller.rb

```diff
@@ -1,9 +1,6 @@
-# Handles static pages
+# Controller for the DAHLIA homepage.
 class HomeController < ApplicationController
-  def index
-  end
-
-  def plus_housing
```

Comment:
I tested this to see if it causes a problem, but it looks like we still redirect as expected if you go to /mohcd-plus-housing